hana/ha_cluster.sls

```diff
@@ -38,6 +32,24 @@ install_SAPHanaSR:
       - SAPHanaSR-doc
 {% endif %}
 
+{% set sr_hook_multi_target = hook_path + '/SAPHanaSrMultiTarget.py' %}
+{% set sr_hook = hook_path + '/SAPHanaSR.py' %}
+{% set sustkover_hook = hook_path + '/susTkOver.py' %}
+{% set suschksrv_hook = hook_path + '/susChkSrv.py' %}
+
+{% set sustkover_hook_enabled = False %}
```

Comment:
This looks a bit strange.
Isn't any option to set a default value if the value doesn't exist?
I mean, i find it rare this `if` condition usage. It is a pretty basic feature
Something like:

```
{% set sustkover_hook_enabled = hana.ha_dr_sustkover_enabled|default(False) %}
```

Reply:
Let me double check if the approach you mention (using `default()`) works now... My fist attempt was to use a Dict like `hana.ha_dr.sustkover.enabled` ...
And I had issues if one of the "levels" did not exist. As we do not do any yaml validation we have issues like this :(
Now that I use a flat variable structure, this could work... Will come back.

Reply:
I did quite some testing now passing `default()`, with several inputs and it works for me.
Pushed the changes you suggested.